osgar/drivers/oak_camera.py

```diff
@@ -29,11 +30,15 @@ def __init__(self, config, bus):
         self.input_thread = Thread(target=self.run_input, daemon=True)
         self.bus = bus
 
-        self.bus.register('depth', 'color', 'orientation_list')
+        self.bus.register('depth', 'color', 'orientation_list', 'detections',
+                          # *_seq streams are needed for output sync amd they are published BEFORE payload data
+                          'depth_seq', 'color_seq', 'detections_seq')
```

Comment:
This (`*_seq`) could be better in one stream.

Reply:
yes, still not sure about the transition, so I would maybe start with this one working and review it once we collect more experience? (yes, I am trying to postpone any changes to the larger/old branch - my fault)

Reply:
I am not sure in this moment. I dont want to block this so feel free merge it. I can change my opinion in future, but i dont like it now. Yes, more experiences will help.

Reply:
OK, thanks - let's see once say the "follow road" is integrated ...